src/solvers/smt/smt_conv.cpp

```diff
@@ -2275,9 +2275,19 @@ expr2tc smt_convt::get_by_ast(const type2tc &type, smt_astt a)
   }
 
   default:
-    msg.error(fmt::format(
-      "Unimplemented type'd expression ({}) in smt get", type->type_id));
-    abort();
+    if(options.get_bool_option("non-supported-models-as-zero"))
```

Comment:
Shall you add a negation here?

If the user sets `--non-supported-models-as-zero`, this method will return true.